SagaFlow.UI/src/lib/CommandStatuses.svelte

```diff
@@ -0,0 +1,259 @@
+                />
+            </td>
+        </tr>
+    </tfoot>
+</table>
+
+<style lang="scss">
+    .command-statuses {
+      
+      .command-statuses-search-container {
+        text-align: var(--sf-command-status-search-alignment, left);
+        
+        .command-statuses-search {
+          padding: var(--sf-command-status-search-padding, initial);
+          font-size: var(--sf-command-status-search-font-size, initial);
+          width: var(--sf-command-status-search-width, auto);
+          border: var(--sf-command-status-search-border, revert);
+        }
+      }
+      
+      td.command-status-col, th.command-status-col {
+        padding: var(--sf-command-status-cell-padding, 1rem 0.25rem);
+      }
+      
+      td.command-status-col {
+        color: var(--sf-command-status-cell-color, inherit);
+        font-style: var(--sf-command-status-cell-font-style, inherit);
+        font-weight: var(--sf-command-status-cell-font-weight, inherit);
+        font: var(--sf-command-status-cell-font, inherit);
+      }
+
+      th.command-status-col {
+        color: var(--sf-command-status-header-color, inherit);
+        font-style: var(--sf-command-status-header-font-style, inherit);
+        font-weight: var(--sf-command-status-header-font-weight, inherit);
+        font: var(--sf-command-status-header-font, inherit);
+      }
+      
+      .command-status-row {
+        i::after {
+          font-style: var(--sf-command-status-icon-font-style, normal) !important;
+        }
+        
+        .icon-col {
+          display: var(--sf-command-status-icon-column-display, table-cell);
+          width: var(--sf-command-status-icon-column-width, auto);
+        }
+        
+        .command-col {
+          display: var(--sf-command-status-command-column-display, table-cell);
+          width: var(--sf-command-status-command-column-width, auto);
+        }
+        
+        .name-col {
+          display: var(--sf-command-status-name-column-display, table-cell);
+          width: var(--sf-command-status-name-column-width, auto);
+        }
+        .user-col {
+          display: var(--sf-command-status-user-column-display, table-cell);
+          width: var(--sf-command-status-user-column-width, auto);
+        }
+        
+        .start-col {
+          display: var(--sf-command-status-start-column-display, table-cell);
+          width: var(--sf-command-status-start-column-width, auto);
+        }
+        
+        .finish-col {
+          display: var(--sf-command-status-finish-column-display, table-cell);
+          width: var(--sf-command-status-finish-column-width, auto);
+        }
+        
+        .progress-col {
+          display: var(--sf-command-status-progress-column-display, table-cell);
+          width: var(--sf-command-status-progress-column-width, auto);
+        }
+
+        &-selected {
+          font-weight: var(--sf-command-status-selected-font-weight, bold) !important;
```

Comment:
Why are we using Important here? My go to rule is never use important as it prevents more specific css selectors from being able to override. Most cases can be fixed with a more specific selector

Reply:
because this is a css class to indicate selection for a table row, it is also at the same level as the table row selector for a successful job or an errored job.  In this case the only way to give '-selected' more precent was to make it important.